WordPress/WordPressTest/WordPressTest-Bridging-Header.h

```diff
@@ -6,3 +6,4 @@
 #import "TestContextManager.h"
 #import "TestingAppDelegate.h"
 #import <OHHTTPStubs/OHHTTPStubs.h>
+#import "BlogService.h"
```

Comment:
If you add `@testable import WordPress` in `ReaderReblogActionTests`, this shouldn't be necessary.

Reply:
That's right. Removed